### PR TITLE
Feature/live display

### PR DESCRIPTION
**Pull request by DanielmTheDev**
I also sneakily removed the progress image in a separate commit.

Gif shows ellipsis and therefore that this custom config works:

Animation

### CODE REVIEW

File: src/spectrecoff-cli/Program.fs [hunk 1]
```diff
@@ -260,6 +260,21 @@ let main argv =
                     .WithDescription("Shows the documentation for the layout module.")
                     |> ignore
             ) |> ignore
+        
+        config
+            .AddBranch("live-display", fun(branchConfig: IConfigurator<LiveDisplaySettings>) ->
+                branchConfig
+                    .AddCommand<LiveDisplayExample>("example")
+                    .WithAlias("e")
+                    .WithDescription("Shows examples of live display module.")
```
Comment (EluciusFTW) [inline src/spectrecoff-cli/Program.fs:269]: deja-vu? "... of the live ...", please.

File: src/spectrecoff/LiveDisplay.fs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿[<AutoOpen>]
+module SpectreCoff.LiveDisplay
+
+open System.Threading.Tasks
+open Spectre.Console
+
+type Configuration =
+    { AutoClear: bool Option
```
Comment (EluciusFTW) [inline src/spectrecoff/LiveDisplay.fs:8]: Not a big fan of bool Options in general. I mean, it's either true or false, right?

**Review by EluciusFTW — APPROVED** [review]

File: src/spectrecoff-cli/Program.fs (hunk 1 above)
Comment (DanielmTheDev) [reply src/spectrecoff-cli/Program.fs]: Slow learner :(

File: src/spectrecoff/LiveDisplay.fs (hunk 2 above)
Comment (DanielmTheDev) [reply src/spectrecoff/LiveDisplay.fs]: That is a very good point actually, makes little sense :) I will keep it in mind

**Pull request merged**